simpleca/Cred_Mgmt_SimpleCA_Feature_Summary_Frag.adoc

```diff
@@ -2,19 +2,14 @@
 [[simpleca-featuresummary]]
```

Comment:
For this file it would actually be easy to keep the old features (i.e. in a "Features new in GT 6" section). But still why duplicate that information if it's kept or will be kept in the GCT/TG6.0 documentation tree?